js/js.tests/build.gradle.kts

```diff
@@ -145,6 +146,25 @@ projectTest("jsIrTest", true) {
     setUpJsBoxTests(jsEnabled = false, jsIrEnabled = true)
 }
 
+projectTest("jsEs6IrTest", true) {
+    systemProperty("kotlin.js.ir.pir", "false")
+    systemProperty("kotlin.js.ir.es6", "true")
+
+    dependsOn(":dist")
+    dependsOn(":kotlin-stdlib-js-ir:generateFullRuntimeKLib")
+    dependsOn(":kotlin-stdlib-js-ir:generateReducedRuntimeKLib")
+    dependsOn(":kotlin-stdlib-js-ir:generateKotlinTestKLib")
+
+    exclude("org/jetbrains/kotlin/js/test/wasm/semantics/*")
+    exclude("org/jetbrains/kotlin/js/test/ir/semantics/*")
```

Comment:
Why?

Reply:
It's runs tests with TargetBackend.JS_IR
Ir tests with TargetBackend.JS_IR_ES6 here: `org/jetbrains/kotlin/js/test/es6/semantics/*`